### PR TITLE
pushing changes on payment, compatibility, profile screens

### PR DESCRIPTION
**Pull request by ReactwithHemanth**
Latest updates :
package and subscription completed
courosal card for package completed
compatibility 3 tabs  slider completed

### CODE REVIEW

File: src/Components/CustomComponents.js [hunk 1]
```diff
@@ -285,3 +285,37 @@ export const Backdrop = ({movies, scrollX}) => {
...
+        source={require('../Assets/error/error.png')}
+        style={styles.ErrorView1}
+        resizeMode="contain"
+      />
+      {/* <Image
+        source={require('../Assets/loading/circleBg.png')}
+        style={{width: width / 2, height: height + 30, alignSelf: 'center'}}
+        resizeMode="contain"
+      /> */}
+
```
Comment (vinay-sayone) [inline src/Components/CustomComponents.js]: Remove comments if not needed.

File: src/Screens/Footer/Footer.js [hunk 2]
```diff
@@ -78,30 +76,44 @@ const Footer = props => {
...
+  //     {FooterVisibility && (
+  //       <View style={[styles.footer]}>
+  //         {item.map((item, idx) => {
+  //           return <ItemRender key={idx} value={item} />;
+  //         })}
+  //       </View>
+  //     )}
+  //   </>
+  // );
 };
```
Comment (vinay-sayone) [inline src/Screens/Footer/Footer.js:102]: Check these comments as well. If not needed remove it. Make sure to remove all the comments before committing changes.

**Review by vinay-sayone — CHANGES_REQUESTED** [review]
Check comments and make changes.

File: src/Screens/Footer/Footer.js (hunk 2 above)
Comment (ReactwithHemanth) [reply src/Screens/Footer/Footer.js:102]: Removed the comments from screens

File: src/Components/CustomComponents.js (hunk 1 above)
Comment (ReactwithHemanth) [reply src/Components/CustomComponents.js]: removed unwanted comments

**Comment by ReactwithHemanth** [comment]
**changed the un wanted comments**

fixed react-native Android App crashes on start: 
`FacebookSdk.sdkInitialize. The sdk has not been initialized `

**Review by vinay-sayone — APPROVED** [review]
approved

**Pull request merged**